### DISCUSSION TITLE
Kaplan Meier Filter censored right data

### DISCUSSION THREAD

**Discussion by jmarchand1** [CamDavidsonPilon/lifelines#1558] [Q&A]
Hello,

I don't see in the code or when i use the KaplanMeier Fillter function, the impact of censored right data after the next death event?
S(ti/ti-1) = (Ni - Di) / Ni = 1 - Di / Ni
Ni = (number live  ti-1) - (number  death à ti) - (number censored between ti-1 et ti)

Could you please explain me.

**Comment by CamDavidsonPilon** [comment]
The Di and Ni are functions of time. A censored death will change Ni, and a death even will change both Ni and Di.

**Reply by jmarchand1** [reply]
Thank you for your answer. 
I am surprise to not see on the plot, an impact on the next time step after several censure compare to a  time interval where there is no censure.
Several censure should decrease more sharply the next time interval probability ?

**Reply by CamDavidsonPilon** [reply]
> Several censure should decrease more sharply the next time interval probability ?

Not necessarily. If deaths are nil that interval, nothing changes
